NewSpring/ApollosProject
# [DEV-3665] Add Prayer Menu

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a 4th nav bar item, and creates the initial menu structure on the main prayer tab. 

### How do I test this PR?
1. Run this branch locally.
2. You should see a new 4th nav bar item (praying man icon) in between "Discover" and "Connect".
3. Tap that icon.
4. You should be taken to the prayer tab. Here's what you should see:
<img width="545" alt="Screenshot 2019-04-16 19 57 10" src="https://user-images.githubusercontent.com/3229463/56251499-dad5c080-6081-11e9-8682-e1a7a65d0eed.png">
5. The menu is a horizontal scroll with 5 options:

- My Saved Prayers
- My Church
- My Campus
- My Community
- My Prayers

6. By tapping on each one of these items you should see a different description appear beneath it. 

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3665](https://newspring.atlassian.net/browse/DEV-3665)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
